source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -716,6 +819,29 @@ Why is waitQueueTimeoutMS optional for some drivers?
 
 We are anticipating eventually introducing a single client-side timeout mechanism, making us hesitant to introduce another granular timeout control. Therefore, if a driver/language already has an idiomatic way to implement their timeouts, they should leverage that mechanism over implementing waitQueueTimeoutMS.
 
+Why must ensuring minPoolSize require the use of a background thread or async I/O?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Without the use of a background thread, minPoolSize is ensured during
+checkOut. `Connections <#connection>`_ may not be checked into the pool before
+being established, however, so establishment also happens during
```

Comment:
Connections cannot be checked in before they are checked out, therefore  it does not make sense to say  a connection is checked in before being established. I suggest simply saying that a connection cannot be checked out prior to being established. 

Separately, I would say this sentence is not needed to clarify why background thread or async i/o is required - jumping from first to third sentence seems perfectly fine to me. The issue is not waiting for one connection to be established but waiting for a large number of connections to be established.

Reply:
:+1: